tm2py/components/network/transit/transit_assign.py

```diff
@@ -2231,13 +2231,41 @@ def _journey_levels(self) -> EmmeTransitJourneyLevelSpec:
 
             elif self.name == "WLK_TRN_WLK":
                 new_journey_levels = copy.deepcopy(journey_levels)
```

Comment:
Adding some annotation of the changes. The `journey_levels` is a list of dictionaries, with each dictionary defining a journey level. Due to the complexity of the fare system, the `journey_levels` (or states) are specific to different fares-paid states, which are created programmatically. (In Task Order 5, the programmatic approach will replaced with something simpler). 

What's being done in the below code snippet is adding a journey level. This state will be walking prior to boarding. As a first step, we are, for each journey level, adding the ability to transition to all the existing states (i.e., I can go from the walking prior to boarding state to each of the transit states).

fyi @lmz, @FlaviaTsang